### PR TITLE
ENYO-5228: Fix to scroll properly with all disabled items in VirtualList

### PR DESCRIPTION
**Pull request by ybsung**
### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

Even though pressing a page up/down key in VirtualList with all disabled items, the list does not scroll.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

We tried to cache the value of the scroll distance when mounting a VirtualList.
But at that time after updating React version, the list's client height equals 0 when `componentDidMount` called, then the cached value also equals 0 and finally the list does not scroll.

Sometimes the client size and scroll direction could be changed dynamically. So I think that it would be better not to cache it instead of checking if they are changed or not.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

ENYO-5228

### Comments

Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### CODE REVIEW

**Comment by ybsung** [comment]
I'll update `CHANGELOG.md` later after being reviewed.

**Comment by 0x64** [comment]
LGTM

File: packages/ui/Scrollable/Scrollable.js [hunk 1]
```diff
@@ -600,7 +596,8 @@ class ScrollableBase extends Component {
 		const
 			bounds = this.getScrollBounds(),
 			canScrollVertically = this.canScrollVertically(bounds),
-			pageDistance = isPageUp(keyCode) ? (this.pageDistance * -1) : this.pageDistance;
+			direction = isPageUp(keyCode) ? -1 : 1,
```
Comment (MikyungKim) [inline packages/ui/Scrollable/Scrollable.js]: Let's not make `direction` here.

**Comment by MikyungKim** [comment]
Please add ChangeLog.

**Comment by MikyungKim** [comment]
LGTM

**Pull request merged**